content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst

```diff
@@ -4,185 +4,229 @@ Process receipts and deliveries in two steps
+and a subsequent delivery order (DO).
 
-In the :menuselection:`Sales` application, create a new quote by clicking :guilabel:`New`. Select
-(or create) a :guilabel:`Customer`, add a storable :guilabel:`Product` to the order lines, and then
-click :guilabel:`Confirm`.
+Create sales order
+------------------
 
-After confirming the :abbr:`SO (Sales Order)`, a :guilabel:`Delivery` smart button will appear in
-the top, above the :abbr:`SO (Sales Order)` form. Click the :guilabel:`Delivery` smart button to
-reveal the associated receipt.
+To create a |SO|, navigate to the :menuselection:`Sales app`, and click :guilabel:`New`. This
+opens a blank sales quotation form.
 
-.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-delivery-button.png
+Add a customer in the :guilabel:`Customer` field. Then, fill out the various fields on the sales
+quotation form, as necessary.
+
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-new-sales-order.png
    :align: center
-   :alt: Delivery smart button on sales order form.
+   :alt: Filled out new sales order form.
 
-.. tip::
-   Sales order receipts can also be found in the :menuselection:`Inventory` application. In the
-   :guilabel:`Overview` dashboard, click the :guilabel:`# To Process` smart button in the
-   :guilabel:`Pick` Kanban card.
+Under the :guilabel:`Product` tab, click :guilabel:`Add a product`, and select a product to add to
+the sales order quotation.
+
+Once ready, click :guilabel:`Confirm`. This moves the quotation to the :guilabel:`Sales Order`
+stage.
+
+Once the |SO| is confirmed, a :guilabel:`Delivery` smart button appears at the top of the form.
+Clicking the smart button opens the warehouse delivery (WH/OUT) form.
+
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-delivery-button.png
+   :align: center
+   :alt: Delivery smart button on validated sales order form.
 
-Process the picking
--------------------
+Process picking
+---------------
 
 Once the sales order is confirmed, a picking order is generated and ready to process.
 
-Navigate to the :menuselection:`Inventory app`, and locate the :guilabel:`Pick` task card on the
-:guilabel:`Inventory Overview` dashboard. This reveals a list of all pickings to process.
+To complete the picking, navigate to the :guilabel:`Inventory app`, and locate the :guilabel:`Pick`
+task card on the :guilabel:`Inventory Overview` dashboard.
+
+Click the :guilabel:`# To Process` button on the task card. This reveals a list of all pickings to
+process.
 
 Click on the picking (WH/PICK) operation associated with the sales order to reveal the picking
 order.
 
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-picking-form.png
+   :align: center
+   :alt: Picking order form for products included in sales order.
+
 Click :guilabel:`Check Availability` on the picking order form. If the product is in stock, Odoo
```

Comment:
```suggestion
Click :guilabel:`Check Availability` on the picking order form. If the product is in stock, Odoo
```

I'm not showing this on the pick operation

Reply:
In 17.2 this shows on the form

Reply:
Yeah, I'm in agreement with Tim on this once again. I even double-checked on a 17.2 runbot, and still wasn't seeing a 'Check Availability' option on the picking order form.
![Screenshot 2024-05-06 103254](https://github.com/odoo/documentation/assets/73958186/5a0b29ff-af39-47aa-9db9-9dc4487e61b0)

Reply:
Turns out the check availability button was showing in Master, not 17.2... so I'm making the necessary changes here